Tests/WorkspaceTests/InitTests.swift

```diff
@@ -75,33 +126,30 @@ final class InitTests: XCTestCase {
             try initPackage.writePackageStructure()
 
             // Not picky about the specific progress messages, just checking that we got some.
-            XCTAssertGreaterThan(progressMessages.count, 0)
-            
+            #expect(progressMessages.count > 0)
+
             // Verify basic file system content that we expect in the package
             let manifest = path.appending("Package.swift")
-            XCTAssertFileExists(manifest)
+            expectFileExists(at: manifest)
             let manifestContents: String = try localFileSystem.readFileContents(manifest)
             let version = InitPackage.newPackageToolsVersion
             let versionSpecifier = "\(version.major).\(version.minor)"
-            XCTAssertMatch(manifestContents, .prefix("// swift-tools-version:\(version < .v5_4 ? "" : " ")\(versionSpecifier)\n"))
+            expectMatch(manifestContents, .prefix("// swift-tools-version:\(version < .v5_4 ? "" : " ")\(versionSpecifier)\n"))
 
-            XCTAssertEqual(try fs.getDirectoryContents(path.appending("Sources").appending("Foo")), ["Foo.swift"])
-            await XCTAssertBuilds(
-                path,
-                buildSystem: .native,
-            )
+            #expect(try fs.getDirectoryContents(path.appending("Sources").appending("Foo")) == ["Foo.swift"])
+            await expectBuilds(path, buildSystem: .native)
```

Comment:
I understand we are waiting for  #8993  to be merged, so my this comment may be irrelevant, but can we replace `expectBuilds` with the following 

```
executeSwiftBuild(
    path,
    configuration: config,
    buildSystem: .native
)
```

Doing so will require having the pass `BuildConfiguraiton.allCases` as an argument.

See `Tests/FunctionalTests/ModuleAliasingFixtureTests.swift` in #8993 for an example migration.